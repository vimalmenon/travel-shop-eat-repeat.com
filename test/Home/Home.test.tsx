import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Home from '../../pages';

describe('pages > <Home> Component', () => {
  describe('Snapshot Testing', () => {
    it('<Home>', () => {
      const wrapper = shallow(
        <Home
          navigation={[]}
          config={{
            appName: 'App Name',
            appDescription: 'App Description',
            titleSeparator: '|',
            version: '0.0.0',
            mode: 'D',
          }}
          navigationList={{}}
        />
      );
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
