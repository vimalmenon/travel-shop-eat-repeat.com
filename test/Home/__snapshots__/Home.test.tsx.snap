// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages > <Home> Component Snapshot Testing <Home> 1`] = `
<div
  className="container"
>
  <HomePage />
</div>
`;
